rpm/podman.spec

```diff
@@ -279,6 +284,55 @@ rm -f %{buildroot}%{_mandir}/man5/docker*.5
 install -d -p %{buildroot}/%{_datadir}/%{name}/test/system
 cp -pav test/system %{buildroot}/%{_datadir}/%{name}/test/
 
+# The compiled set of ginkgo tests, and a helper script
+install -d -p %{buildroot}/%{_datadir}/%{name}/test/e2e
+cp test/e2e/e2e.test hack/podman-registry %{buildroot}/%{_datadir}/%{name}/test/e2e
+
+# Files and subdirectories used by those tests
+for testfiles in certs deny.json policy.json redhat_sigstore.yaml registries.conf; do
+    cp -pav test/$testfiles %{buildroot}/%{_datadir}/%{name}/test/e2e/
+done
+for subdir in build cdi config quadlet sign testdata; do
+    cp -pav test/e2e/$subdir %{buildroot}/%{_datadir}/%{name}/test/e2e/
+done
+
+# Invoking ginkgo is complicated. Make it easier for end users.
+cat >%{buildroot}/%{_datadir}/%{name}/test/e2e/run-tests <<EOF
+#!/bin/bash
+#
+# Possible options:
+#
+#   --ginkgo.focus "blah blah"
+#   --ginkgo.flake-attempts 3
+#   --ginkgo.parallel.total 3
+#   --ginkgo.randomize-all
+#
+
+# /tmp is probably too small for all we do
+export TMPDIR=/var/tmp
+
+# Fetch images only once. Important when rerunning e2e tests.
+export PODMAN_TEST_IMAGE_CACHE_DIR=/var/tmp/ginkgo
+mkdir -p \$PODMAN_TEST_IMAGE_CACHE_DIR
+
+# The executables we're testing are the ones delivered in RPM
+export PODMAN_BINARY=%{_bindir}/%{name}
+export PODMAN_REMOTE_BINARY=%{_bindir}/%{name}-remote
+export QUADLET_BINARY=%{_libexecdir}/%{name}/quadlet
+
+# Where subtests will find files they need
+export PODMAN_INTEGRATION_ROOT=%{_datadir}/%{name}/test/e2e
+
+# Many tests rely on files relative to cwd
+cd \$PODMAN_INTEGRATION_ROOT/
+
+# Needed for podman-registry script
+export PATH=\$PATH:\$PODMAN_INTEGRATION_ROOT
+
+exec ./e2e.test --ginkgo.trace --ginkgo.v "\$@"
```

Comment:
without proper build tags this has low changes of success, or maybe these need to bet set at compile time already?

Reply:
The build tags are set at compile time; I've just pushed a change that should take care of that. I did not understand that yesterday.